codesign/lib/src/globals.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: unnecessary_string_escapes
+
+import 'package:args/args.dart';
+
+class ConductorException implements Exception {
+  ConductorException(this.message);
+
+  final String message;
+
+  @override
+  String toString() => 'Exception: $message';
```

Comment:
I don't think you need to override `toString()` here.

Reply:
umm I did a quick test on removing it. I think overriding has the effect of showing 
`Exception: i am a test message` instead of `Instance of 'ConductorException'`, which is a bit more verbose.